README.md

```diff
@@ -388,27 +425,15 @@ Class | Method | HTTP request | Description
 
 ## Documentation for Authorization
```

Comment:
The `Documentation for Authorization` part should remain as before. Also, please add back the `Support and Feedback` section